SConstruct

```diff
@@ -121,6 +121,10 @@ AddOption('--ignore-style', action='store_true',
           help='Disable style checking hooks')
 AddOption('--linker', action='store', default=None, choices=linker_options,
           help=f'Select which linker to use ({", ".join(linker_options)})')
+AddOption('--mold-path', action='store', default='/usr/bin/mold',
+          help='Path to the binary of mold, the linker. For GCC, the path must'
+               ' be a relative path. For clang, the path can be an absoblute'
```

Comment:
We don't support the mold linker for clang, so it doesn't matter. 

Are you sure for GCC it needs to be a relative path, because the current paths specified, such as `conf.CheckLinkFlag('-B/usr/libexec/mold'))`, appear absolute me.

Reply:
Yeah, the part about relative part is not true. I'll modify it.